developer-docs-site/docs/tutorials/run-a-local-testnet.md

```diff
@@ -7,19 +7,23 @@ import BlockQuote from "@site/src/components/BlockQuote";
 
 # Run a Local Testnet
 
-You can run a local testnet to test and develop against an Aptos Blockchain. This network runs independently of the Aptos ecosystem and is for testing and development purposes only.
+You can run a local testnet of the Aptos Blockchain. This network runs independently of the Aptos ecosystem and is primarily for testing and development purposes.
```

Comment:
maybe `solely for testing and development`.  It is primarily for testing and development, but it shouldn't be for any sort of production use case, so I'd be more specific